lib/cartopy/crs.py

```diff
@@ -1047,19 +1048,45 @@ def __init__(self, central_longitude=-96.0, central_latitude=39.0,
                         ('lat_0', central_latitude),
                         ('x_0', false_easting),
                         ('y_0', false_northing)]
-        if secant_latitudes is not None:
-            proj4_params.append(('lat_1', secant_latitudes[0]))
-            proj4_params.append(('lat_2', secant_latitudes[1]))
+        if secant_latitudes and standard_parallels:
+            raise TypeError('standard_parallels replaces secant_latitudes.')
```

Comment:
That is a reasonable question. The other option was a ValueError, but it is questionable whether that is the correct type too. The only thing that got me off the fence was that, once the keyword is removed, using it will cause a TypeError:

```
>>> def foo(a=1):
...       pass

>>> foo(b=1)
File "<stdin>", line 1, in <module>
TypeError: foo() got an unexpected keyword argument 'b'
```